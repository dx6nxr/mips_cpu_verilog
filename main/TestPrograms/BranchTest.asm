lui $1, 0x0000
ori $1, 0xffff
beq $1, $1, labl
lui $2, 0x0000
ori $2, 0x0001
labl:
lui $3, 0x0000
ori $3, 0xffff
