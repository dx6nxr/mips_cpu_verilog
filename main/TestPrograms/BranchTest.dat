3c01ffff
3421ff9c
10210002
3c020000
34420001
3c030000
3463ffff