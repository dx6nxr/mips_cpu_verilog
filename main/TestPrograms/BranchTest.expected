@1 ffffff9c
@3 ffff